src/Bicep.Core/Syntax/SyntaxVisitor.cs

```diff
@@ -139,6 +144,10 @@ public virtual void VisitSkippedTriviaSyntax(SkippedTriviaSyntax syntax)
             }
         }
 
+        public virtual void VisitMissingSyntax(MissingSyntax syntax)
+        {
```

Comment:
Should probably still call Accept, no? Even if we don't do anything with it

Reply:
Will be deleted when I delete the missing syntax class.

Reply:
It's gone.